docs/content/preview/drivers-orms/orms/ysql-pg.md

```diff
@@ -1,13 +1,13 @@
 ---
```

Comment:
This one isn't really like the others. Does it belong here?

Reply:
nice catch , yes this one is not a REST example.